lib/spack/spack/cmd/env.py

```diff
@@ -523,6 +524,34 @@ def env_revert(args):
     tty.msg(msg.format(manifest_file))
 
 
+def env_generate_makefile_setup_parser(subparser):
+    pass
+
+
+def env_generate_makefile(args):
+    env = ev.active_environment()
+    if not env:
+        tty.die("required env")
+    hash_to_deps_hashes = dict()
+    hash_to_spec = dict()
```

Comment:
can you use `{}`?  I guess I don't care *all* that much but we use it most other places.